app/views/events/_past.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="m-5">
+<% if Event.previous.nil? %>
+  <%= "There's not previous events" %>
+<% else %>
+<% Event.previous.each do |inv| %>
```

Comment:
You should make use of a variable instead. Check the comment of the event index controller.